org.eclipse.winery.lsp/src/test/resources/ToscaFileTest.yaml

```diff
@@ -0,0 +1,72 @@
+##########################################################################
+# This file contains TOSCA Simple Profile artifact type
+# definitions.
+##########################################################################
+
+tosca_definitions_version: tosca_2_0
+
+metadata:
+  template_name: artifact_types.yaml
+  template_author: Oasis Open
+  template_version: 2.0
+
+description: >
+  TOSCA Simple Profile artifact type definitions
+
+profile: org.oasis-open.tosca.simple:2.0
+
+artifact_types:
+
+  Root:
+    description: >
+      The TOSCA base artifact type from which all other normative
+      TOSCA artifact types derive.
+
+  File:
+    description: >-
+      The File artifact type is used when an artifact definition needs
+      to have its associated file simply treated as a file without any
+      special handling.
+    derived_from: Root
+
+  Deployment:
+    description: >-
+      The Deployment artifact type represents a binary packaging of an
+      application or service that is used to install/create or deploy
+      it as part of a node’s lifecycle.
```

Comment:
Where did you get the text from? Please state the source!

Reply:
here: https://github.com/oasis-open/tosca-community-contributions/blob/master/profiles/org.oasis-open/simple/2.0/artifact_types.yaml

Reply:
Quick fix: Move the file to a sub directory "tosca-community-contributions" and ad a README.md there

Better fix: Use `git submodule` ti include that repository in `test/resources`. This makes the source of the files transparent and also eases updates. The drawback is that contributors need to use git's submodules.